component-classes/index.js

```diff
@@ -555,4 +555,16 @@ export const backwardsCompatibleClasses = {
   radioDisabled: 'before:i-bg-$color-radio-background-disabled before:i-bg-$color-checkbox-background-disabled peer-checked:before:i-border-$color-radio-border-selected-disabled', //replaced in v1.5.0
   modalTitle: 'h-40 sm:h-48 items-center', // replaced by min-h-40 sm:min-h-48 items-start
   pillClose: 'pt-4 pb-6 text-m!' //replaced by py-10
+};
+
+export const pagination = {
+  currentPage: 'block md:hidden p-8 font-bold',
+  icon: 'flex w-16 h-16',
+  containerNav: 'flex items-center justify-center space-x-8 p-8',
+  heading: 'sr-only',
+  pages: 'hidden md:block i-text-$color-pagination-text',
+  nextPage: 'flex items-center',
+  firstPageButton: 'md:hidden',
+  firstPageLabel: 'font-bold',
+  active: 'i-bg-$color-pagination-background-active i-text-$color-pagination-text-active',
```

Comment:
@magnuh Not sure if these are the correct colors tho

Reply:
Since these are new component classes we might as well just use the semantic tokens from the start. 🙂 (Since there is no difference between Tori and Finn for this component)
 ```suggestion
  active: 's-bg-primary s-text-inverted',
```

Reply:
But will this cause issues in the future if any market decides to use a different color than primary?

Reply:
Also `s-bg-primary` doesn't generate a class, it should be called something else?

Reply:
At least there is a `s-bg-primary` here https://warp-ds.github.io/css-docs/background-color 😅